Makefile

```diff
@@ -0,0 +1,97 @@
+BINPATH ?= build
+
+GREEN  := $(shell tput -Txterm setaf 2)
+YELLOW := $(shell tput -Txterm setaf 3)
+WHITE  := $(shell tput -Txterm setaf 7)
+CYAN   := $(shell tput -Txterm setaf 6)
+RESET  := $(shell tput -Txterm sgr0)
+
+BUILD_TIME=$(shell date +%s)
+GIT_COMMIT=$(shell git rev-parse HEAD)
+VERSION ?= $(shell git tag --points-at HEAD | grep ^v | head -n 1)
+
+LDFLAGS = -ldflags "-X main.BuildTime=$(BUILD_TIME) -X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)"
+
+.PHONY: all
+all: delimiter-AUDIT audit delimiter-LINTERS lint delimiter-UNIT-TESTS test delimiter-COMPONENT_TESTS test-component delimiter-FINISH ## Runs multiple targets, audit, lint, test and test-component
+
+.PHONY: audit
+audit: ## Audits and finds vulnerable dependencies
+	go list -json -m all | nancy sleuth
+
+.PHONY: build 
+build: Dockerfile ## Builds ./Dockerfile image name: scrubber
+	docker build -t scrubber .
+
+.PHONY: build-bin
+build-bin: ## builds bin
+	go build -tags 'production' $(LDFLAGS) -o $(BINPATH)/scrubber
+
+.PHONY: clean
+clean: ## Removes /bin folder
+	rm -fr ./build
+	rm -fr ./vendor
+
+.PHONY: convey
+convey: ## Runs Convey tests
+	goconvey ./...
+	
+.PHONY: debug
+debug: ## Runs the api locally in debug mode
+	go build -tags 'debug' $(LDFLAGS) -o $(BINPATH)/dp-nlp-search-scrubber
+	HUMAN_LOG=1 DEBUG=1 $(BINPATH)/dp-nlp-search-scrubber
+
+.PHONY: delimiter-%
+delimiter-%:
+	@echo '===================${GREEN} $* ${RESET}==================='
+
+.PHONY: fmt 
+fmt: ## Formats the code using go fmt and go vet
+	go fmt ./...
+	go vet ./...
+
+.PHONY: lint 
+lint: ## Automated checking of your source code for programmatic and stylistic errors
+	golangci-lint run ./...
```

Comment:
it fails the lint check - example config here: https://github.com/ONSdigital/dp-topic-api/blob/develop/.golangci.yml